benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars

```diff
@@ -9,7 +9,7 @@ ksa       = "benchmark-ksa"
 
 # Locust service configuration 
 artifact_registry                        = "us-central1-docker.pkg.dev/$PROJECT_ID/ai-benchmark"
-inference_server_service                 = "tgi" # inference server service name
+inference_server_service                 = "http://tgi" # inference server service name
```

Comment:
Huh. Maybe my changes didn't work as expected?: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh#L24

I seem to remember testing both pathways. But maybe not